src/POGOProtos/Enums/StatusCode.proto

```diff
@@ -0,0 +1,13 @@
+syntax = "proto3";
+package POGOProtos.Enums;
+
+enum StatusCode {
+  // normal status codes
+	UNKNOWN_STATUS = 0;
+	OK = 1;
+	// 5x seems to relate to load balance and rate control
+	RATE_LIMITED = 52;
```

Comment:
What does this really mean, rate limited as in "refused because limited"? Better add a note on that